lb_content_resolver/content_resolver.py

```diff
@@ -176,16 +176,18 @@ def add_file_to_index(self, relative_path, format, mtime):
         else:
             status = "added"
 
+        unknown_string = "[unknown]"
         # We've never seen this before, or it was updated since we last saw it.
         if format == "mp3":
-            mdata = mp3.read(file_path, mtime)
+            mdata = mp3.read(file_path, mtime, unknown_string)
         elif format == "flac":
-            mdata = flac.read(file_path, mtime)
+            mdata = flac.read(file_path, mtime, unknown_string)
 
-        mdata["file_path"] = file_path
+        if mdata is not None:
```

Comment:
I also had an error where `mdata` was `None`, I'm not sure why but this works around it.

Reply:
If a file has no tags, I think mdata would be empty. Creating it and setting file_path is the right thing to do here -- I'll fix that in a minute.